examples/with-claims/src/App.js

```diff
@@ -0,0 +1,149 @@
+import { useState } from "react";
+import "./App.css";
+import SuperTokens, { getSuperTokensRoutesForReactRouterDom } from "supertokens-auth-react";
+import EmailPassword from "supertokens-auth-react/recipe/emailpassword";
+import Session, { SessionAuth } from "supertokens-auth-react/recipe/session";
+import { Routes, BrowserRouter as Router, Route } from "react-router-dom";
+import Footer from "./Footer";
+import SessionExpiredPopup from "./SessionExpiredPopup";
+import { RolesClaim } from "./claims/rolesClaim";
+import { ThemeProvider } from "@emotion/react";
+import { CssBaseline } from "@mui/material";
+import { EmailVerifiedClaim } from "./claims/emailVerifiedClaim";
+import { api, TestContextProvider, useTestLogger } from "./test.context";
+import { TestScreen } from "./routes/testScreen.component";
+import { MFAClaim } from "./claims/mfaClaim";
+import { useMemo } from "react";
+import { useNavigate } from "react-router-dom";
+
+export function getApiDomain() {
+    const apiPort = process.env.REACT_APP_API_PORT || 3001;
+    const apiUrl = process.env.REACT_APP_API_URL || `http://localhost:${apiPort}`;
+    return apiUrl;
+}
+
+export function getWebsiteDomain() {
+    const websitePort = process.env.REACT_APP_WEBSITE_PORT || 3000;
+    const websiteUrl = process.env.REACT_APP_WEBSITE_URL || `http://localhost:${websitePort}`;
+    return websiteUrl;
+}
+
+SuperTokens.init({
+    appInfo: {
+        appName: "SuperTokens Demo App", // TODO: Your app name
+        apiDomain: getApiDomain(), // TODO: Change to your app's API domain
+        websiteDomain: getWebsiteDomain(), // TODO: Change to your app's website domain
+    },
+    recipeList: [
+        EmailPassword.init({
+            emailVerificationFeature: {
+                mode: "OFF",
```

Comment:
It's strange that this is given "OFF", but the backend still has email verification as a require default claim.

Reply:
You shouldn't have to set the email verification mode at all on the frontend anymore.

Reply:
Different modes:
- REQUIRED: Means that the default claims would have email verification automatically and redirect
- OPTIONAL: Means that the default claims would have email verification automatically and NOT redirect
- OFF: Means that default claim doesn't have email verification automatically. If the user puts it in the claims themselves for a route, then it should check it and NOT redirect.


One another interpretation could be:
- REQUIRED: Means that the default claims would have email verification automatically and redirect
- OPTIONAL: Means that the default claims would NOT have email verification automatically, but if the user provides it for a route, it SHOULD redirect
- OFF: Means that default claim doesn't have email verification automatically. If the user puts it in the claims themselves for a route, then it should check it and NOT redirect.
----------------------
OR, we should have two config values: `checkForAllRoutes` and `redirectAutomatically`.